src/openvic-simulation/military/Unit.hpp

```diff
@@ -64,8 +64,10 @@ namespace OpenVic {
 	struct LandUnit : Unit {
 		friend struct UnitManager;
 
+		enum struct allowed_cultures_t { ALL_CULTURES, ACCEPTED_CULTURES, PRIMARY_CULTURE };
+
 	private:
-		const bool PROPERTY_CUSTOM_PREFIX(primary_culture, is);
+		const allowed_cultures_t PROPERTY(allowed_cultures);
```

Comment:
I consider renaming it to cultural_restrictions with no_restrictions being the default.
@Hop311 what do you think?

Reply:
I think it's fine the way it is, it's clear what each enum value means, whereas the meaning of `no_restrictions` isn't immediately obvious without looking at the enum's type name.